src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor.Shared.Options;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Formatting
+{
+    public class CodeCleanupTests : FormattingEngineTestBase
+    {
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
```

Comment:
These test should likely have their own trait.

Reply:
fixed